src/intl/messages.js

```diff
@@ -61,7 +61,12 @@ export const messages: { [messageId: string]: MessageType } = {
   bootSequence: 'Boot Sequence',
   cancel: 'Cancel',
   cannotUpdateCloudInitHostname: 'Host name cannot be synchronized with VM name because the VM name is not a valid host name.',
+  cantDownloadConsoleEmptyList: 'There are no consoles assigned to this VM.',
+  cantDownloadConsoleGeneral: "Can't download console:",
```

Comment:
Please rephrase to something more user friendly like: "can't open a graphical console since this is  a headless VM"
Please also add the VM name to the message.

Reply:
> "non RDP Console"...

I would rather not name a key functionality using Windows protocol. It looks as RDP was the default and other consoles were our second choice.

> the RDP seems to be neglected as a separate option, not related to other consoles which is confusing

@bamsalem  Can we treat RDP the same as other consoles? This would remove the source of confusion.

Reply:
@rszwajko 

> > "non RDP Console"...
> I would rather not name a key functionality using Windows protocol. It looks as RDP was the default and other consoles were our second choice.
> 
> > the RDP seems to be neglected as a separate option, not related to other consoles which is confusing
> Can we treat RDP the same as other consoles? This would remove the source of confusion.

I agree. I suggested calling this: "spice/vnc console", "rdp console"

The problem with treating RDP the same as other consoles is that RDP is always enabled for windows based OS vms only, regardless to spice/vnc. So we'll need to add another config value for setting global default console for windows based vms and one for non windows based vms instead of one "ClientModeConsoleDefault". This will be required for account settings solution but I wanted to save that extra config parameter. If you think it's better then let's add this.